orchestration/api/api_clip.py

```
@@ -93,6 +96,32 @@ def add_phrase(request: Request,
 
     return http_clip_server_add_phrase(phrase)
 
+@router.post("/clip/phrases",
+             response_class=PrettyJSONResponse,
+             description="Adds a phrase to the clip server",
+             tags=["clip"])
+@cache(expire=60)  # Cache for 60 seconds, adjust as necessary
+def add_phrase(request: Request):
+    body = request.json()
+    phrase = body.get("phrase")
+
+    if not phrase:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail="Phrase is required")
```

Please return the standard error response from https://github.com/kk-digital/kcg-ml-image-pipeline-frontend/blob/api-doc/API-GUIDELINES.md#identifying-errors